src/components/text-input/index.md.njk

```diff
@@ -126,6 +146,10 @@ Error messages should be styled like this:
 
 {{ example({group: "components", item: "text-input", example: "error", html: true, nunjucks: true, closed: true, size: "s"}) }}
 
+#### Prefixes and suffixes
```

Comment:
Does this heading make sense here? Is there a risk of someone mis-reading this as the main section on prefixes and suffixes?

Would it still make sense if we removed the heading and just had the two examples one after the other?

Reply:
That's my bad, it should be 

> If the input has a prefix or suffix

I'll update it